kikit/substrate.py

```diff
@@ -408,6 +408,20 @@ def biteBoundary(boundary, pointA, pointB, tolerance=fromMm(0.01)):
     return None
 
 
+def collectionToPointList(collection):
```

Comment:
Overall, the change is reasonable. The only thing I am concerned about is the naming of this function. The name is too general and actually misleading. The function doesn't turn a geometry collection into a list of points; instead, it chooses candidates for the closest points for the function `closestIntersectionPoint`. This is confusing. 

I suggest either naming this function appropriately, or actually not introducing it at all. We already have a function `listGeometries` that allows you to iterate through all geometries (even when the source geometry is not a collection), so you can easily implement it in place in `closesIntersectionPoint`

Reply:
Thanks for pointing out `listGeometries()`.  I agree that's a better solution.  But had to wrangle another bug I found before submitting an update.  Sorry for "two bugs in one PR" but they both stand in the way of what I'm trying to do, affecting the same code, so its difficult to split them into two independent PRs.